httpsign/nonce.go

```diff
@@ -23,18 +23,17 @@ func newNonceCache(capacity int, cacheTTL int) (*nonceCache, error) {
 
 // inCache checks if a nonce is in the cache. If not, it adds it to the
 // cache and returns false. Otherwise it returns true.
-func (n *nonceCache) inCache(nonce string) bool {
+func (n *nonceCache) inCache(nonce string) (exists bool, reterr error) {
```

Comment:
nit: the `err` would be more idiomatic.

Reply:
That's a habit I have to differentiate return value from internal `err` variables.  Not necessary in this case.  Fixed.

Reply:
Well, we could enable https://github.com/alexkohler/nakedret for that.